plasmapy/utils/decorators/helpers.py

```diff
@@ -142,3 +142,51 @@ def wrapper(*args, **kwargs):
         wrapper.__signature__ = inspect.signature(f)
 
     return wrapper
+
+
+def add_lite(flite, attrs=None, scope=None):
+    """
+    Decorator to bind lightweight "lite" versions of formulary functions to the full
+    formulary function.
+    """
+    if attrs is None:
+        attrs = []
+    elif scope is None:
+        raise ValueError(
+            f"If 'attrs' are being bound to the decorate function, then 'scope'"
+            f" needs to be defined and contain the objects to be bound.  "
+            f"Setting 'scope=globals()' will typically suffice."
+        )
+
+    def decorator(f):
+        @functools.wraps(f)
+        def wrapper(*args, **kwargs):
+            return f(*args, **kwargs)
+
+        wrapper_doc = inspect.cleandoc(getattr(wrapper, "__doc__", """"""))
+
+        setattr(wrapper, "lite", flite)
+
+        wrapper_doc += "\n\n" + inspect.cleandoc(
+            f"""
+            .. note:: To increase usability of `{f.__name__}` several
+               attributes/functions are manually bound to this function.
+
+               - `{f.__name__}.lite` <--> `~{flite.__module__}.{flite.__name__}`
+            """
+        )
+
+        for bound_name, attr_name in attrs:
+            attr = scope[attr_name]
+
+            setattr(wrapper, bound_name, attr)
+
+            wrapper_doc += (
+                f"\n   - `{f.__name__}.{bound_name}` <--> `~{f.__module__}.{attr_name}`"
+            )
```

Comment:
Looks great [in the docs!](https://plasmapy--1098.org.readthedocs.build/en/1098/api/plasmapy.formulary.parameters.thermal_speed.html#codecell0:~:text=To%20increase%20usability%20of%20thermal_speed%20several,are%20manually%20bound%20to%20this%20function.) :+1: